App/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -875,9 +1010,9 @@ Array [
           >
             About
           </Text>
-        </>
+        </View>
       </View>
     </View>
-  </View>,
-]
+  </RCTSafeAreaView>
+</View>
```

Comment:
This file saw many diff changes but I think it's fine cause it's a general snapshot